.checkov

```diff
@@ -0,0 +1,10 @@
+branch: main
```

Comment:
Name file `.checkov.yaml` With that name you don't have to specify the `config_file` in the workflow.